ColorCoder.h

```diff
@@ -0,0 +1,27 @@
+#ifndef COLORCODER_H
+#define COLORCODER_H
+
+#include <string>
+
+namespace TelCoColorCoder 
+{
+    enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
+    enum MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
+
+    class ColorPair {
+        private:
```

Comment:
Encapsulate ColorPair in another file "ColorPair.h"
Encapsulate Enum's in  *********.h file